test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -259,17 +206,37 @@ public void IrisSdcaMaximumEntropy()
             }).PredictedLabel);
         }
 
-        private string GetTestDatabasePath(string databaseName)
+        /// <summary>
+        /// Non-Windows builds do not support SqlClientFactory/MSSQL databases. Hence, an equivalent
+        /// SQLite database is used on Linux and MacOS builds.
+        /// </summary>
+        /// <returns>Return the appropiate Iris DatabaseSource according to build OS.</returns>
+        private DatabaseSource GetIrisDatabaseSource(string command)
         {
-            return Path.GetFullPath(Path.Combine("TestDatabases", $"{databaseName}.mdf"));
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                return new DatabaseSource(
+                    SqlClientFactory.Instance,
+                    GetMSSQLConnectionString(TestDatasets.irisDb.name),
+                    String.Format(command, $@"""{TestDatasets.irisDb.trainFilename}"""));
```

Comment:
Passing `TestDatasets.irisDb.trainFilename` as `$@"""{TestDatasets.irisDb.trainFilename}"""` is necessary to correctly pass the table name as `\"iris-train\"` in MSSQL.